### PR TITLE
bookings data added

### PR DESCRIPTION
**Pull request by PhilJG**

### CODE REVIEW

File: src/components/Bookings.jsx [hunk 1]
```diff
@@ -11,9 +11,12 @@ function Bookings() {
       rating: 4,
       reviews: 34,
       photo:
-        'https://res.cloudinary.com/dsko6ntfj/image/upload/v1640295026/portal/web%20development%20beginners/05%20Project%20Airbnb/house%2001/house_01_01.png'
+        'https://res.cloudinary.com/dsko6ntfj/image/upload/v1640295026/portal/web%20development%20beginners/05%20Project%20Airbnb/house%2001/house_01_01.png',
+      booking: {
+        startDate: new Date(2024, 8, 22),
```
Comment (ynotrusso) [inline src/components/Bookings.jsx:16]: these should just be strings

File: src/components/HouseCard.jsx [hunk 2]
```diff
@@ -7,6 +7,34 @@ import {
 import { comment } from 'postcss'
 import { Link } from 'react-router-dom'
 
+function BookingComponent({ booking, price }) {
```
Comment (ynotrusso) [inline src/components/HouseCard.jsx:10]: all of this is hardcoded in the db

**Review by ynotrusso — CHANGES_REQUESTED** [review]

**Review by PhilJG — COMMENTED** [review]
What do you want changed here?

File: src/components/Bookings.jsx (hunk 1 above)
Comment (PhilJG) [reply src/components/Bookings.jsx:16]: Don't exactly understand what you want here. I think its one of two possibilities: 

1) Make object properties into strings: But when I try to put " " around these object properties to make a string the formatter takes them off when I save

2) Turn the dates into strings: But when I try new String(new Date(2024, 3, 4)) or String(new Date(2024, 3, 4)) they still come up as an object. 

Not sure which of these you are asking for so more elaboration would be appreciated

**Review by ynotrusso — APPROVED** [review]

**Pull request merged**